src/miniscript/types/mod.rs

```diff
@@ -31,10 +31,12 @@ fn return_none<T>(_: usize) -> Option<T> {
 }
 
 /// Detailed type of a typechecker error
-#[derive(Copy, Clone, PartialEq, Eq, PartialOrd, Ord, Hash, Debug)]
+#[derive(Copy, Clone, PartialEq, Eq, Hash, Debug)]
 pub enum ErrorKind {
     /// Relative or absolute timelock had a time value of 0
     ZeroTime,
+    /// Passed a `z` arguement to a `d` wrapeer when `z` was expected
+    NonZeroDupIf,
```

Comment:
Heh, arguably this should be in a separate commit, but I'll allow it

Reply:
I added it to the other merged PR. Forgot to remove it from here

Reply:
Don't know why this is not creating a conflict

Reply:
If they're exactly the same change git is smart enough to unify them.